lib/parsers/index.ts

```diff
@@ -367,11 +368,9 @@ export function getTargetFrameworksFromProjectConfig(manifestFile) {
 }
 
 export function getTargetFrameworksFromProjectJson(manifestFile) {
-  const targetFrameworksResult: string[] = [];
   return Object.keys(_.get(manifestFile, 'frameworks', {}));
 }
 
 export function getTargetFrameworksFromProjectAssetsJson(manifestFile) {
-  const targetFrameworksResult: string[] = [];
-  return Object.keys(_.get(manifestFile, 'project.frameworks', {}));
+  return Object.keys(_.get(manifestFile, 'targets', {}));
```

Comment:
In snyk-nuget-plugin we get the framework from project.frameworks (https://github.com/snyk/snyk-nuget-plugin/blob/master/lib/nuget-parser/dotnet-core-parser.js#L88)

Why did you change it here?

Reply:
In order to get the deps from the project.assets.json, I needed the targets which have the format ".NETCoreApp,Version=v2.2" instead of "netcoreapp2.2" which is the format from project.frameworks. Is it better to be retrieve the framework from project.frameworks and then do some transfomation to get the targetFramework format I require?
`{
    "version": 3,
    "targets": {
      ".NETCoreApp,Version=v2.2": {
        "log4net/2.0.8": {
          "type": "package",
          "dependencies": {
            "System.AppContext": "4.1.0",`

Reply:
In CLI the required format is `netcoreapp2.2` so that is why it uses project.frameworks, if for SCM we use the other format, then better to not have to create our own targetframework name interpreter.